StockPriceScraper/Program.cs

```diff
@@ -77,8 +78,7 @@ static bool ReadConfigurationFile()
                 outputfilePath = xmldoc.DocumentElement.SelectSingleNode("/config/basics/output-file").InnerText;
 
                 // reading stocks
-                XmlNodeList stocks = xmldoc.DocumentElement.SelectNodes("/config/stocks/stock");
-                foreach (XmlNode stock in stocks)
+                foreach (XmlNode stock in xmldoc.DocumentElement.SelectNodes("/config/stocks/stock"))
```

Comment:
this was not cleaned up because I tend to try out things first for easier debugging. 
When u put all on one line it is harder to debug and spot logical errors.